merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/MerlinParsers.java

```diff
@@ -136,8 +137,8 @@ public JsonValue unparse(final Timestamp value) {
   public static final JsonParser<HasuraAction<HasuraAction.MissionModelArgumentsInput>> hasuraMissionModelArgumentsActionP
       = hasuraActionP(hasuraMissionModelArgumentsInputP)
       .map(Iso.of(
-          untuple(HasuraAction::new),
-          $ -> tuple($.name(), $.input(), $.session())));
+          untuple((name, input, session, requestQuery) -> new HasuraAction<>(name, input, session)),
```

Comment:
🔶 Would it make any sense to add the `request_query` as a field on `HasuraAction`? Or do we prefer to discard it, since we have no use for it?

Reply:
At first that's what I did, but until we have a use for it I think it's probably best to discard it.